azure-pipelines.yml

```diff
@@ -81,19 +81,19 @@ jobs:
         - netcoreapp2.2
         - netcoreapp2.1
 
-  # Windows x64 net461 micro benchmarks
+  # Windows x64 net48 micro benchmarks
   - template: /eng/performance/benchmark_jobs.yml
     parameters:
       osName: windows
       osVersion: RS4
-      kind: micro_net461
+      kind: micro
       architecture: x64
       pool: Hosted VS2017
       queue: Windows.10.Amd64.ClientRS4.DevEx.15.8.Open
```

Comment:
@ViktorHofer @safern do you know which queue|pool has .NET 4.8 SDK installed?

Reply:
right, I assume you are talking about the ref assembly pack? That is available here: https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies.net48/1.0.0-preview.2

Reply:
I want to build and run as 4.8. Is using ref assembly pack going to be enough to achieve that?

Reply:
The ref assembly pack is for compiling on both windows and Unix. If you only care about windows here, you probably don't need that pack as windows always has the latest netframework installed which is updated via windows update. Targeting net48 should just work...

Reply:
In corefx for package testing, since we can't guarantee all our build/test machines have the targeting packs we need installed, we get it via NuGet, to ensure it always works. https://github.com/dotnet/corefx/blob/master/src/pkg/test/frameworkSettings/net/settings.targets#L11
-- even though we only run in Windows.